source/_integrations/google_assistant_sdk.markdown

```diff
@@ -69,6 +69,47 @@ The integration setup will next give you instructions to enter the [Application
 
 {% enddetails %}
 
+## Enable personal results (for advanced users)
+
+If you want to enable personal commands (e.g. "remind me tomorrow at 9 pm to take out the trash") follow the steps below after successfully installing the integration. This is for advanced users since it requires creating an OAuth client ID of Desktop app, running a Python program on your desktop or laptop, copying the resulting credentials to your Home Assistant config files, and likely running an Android emulator.
+
+{% details "Create credentials" %}
+
+1. Navigate to [Google Developers Console > Credentials](https://console.cloud.google.com/apis/credentials)
+2. Select the project you created earlier from the dropdown menu in the upper left corner.
+3. Click Create credentials (at the top of the screen), then select OAuth client ID.
+4. Set the Application type to "Desktop app" and give this credential set a name (like "Home Assistant Desktop Credentials").
+5. Click Create.
```

Comment:
```suggestion
5. Select **Create**.
```
We [favor select over click](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click), because click is more mouse-related.